src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/HealthCertificate/HealthCertificateCellViewModel.swift

```diff
@@ -105,9 +108,9 @@ final class HealthCertificateCellViewModel {
 		} else {
 			return nil
 		}
-	}
+	}()
 
-	var image: UIImage {
+	lazy var image: UIImage = {
```

Comment:
I think if we don't reevaluate the properties every time they are not updating anymore if e.g. the `validityState` of the certificate or the `vaccinationState` of the person changes while the user is on the screen. We probably need publishers then to update the views individually if a reload doesn't trigger those updates anymore.